lib/models/command.js

```diff
@@ -127,14 +127,14 @@ Command.prototype.validateAndRun = function(args) {
 
   if (this.works === 'insideProject' && !this.isWithinProject) {
     return Promise.reject(new SilentError(
-      'You have to be inside an ember-cli project in order to use ' +
-      'the ' + chalk.green(this.name) + ' command.'
+      chalk.yellow('You have to be inside an ember-cli project in order to use the ') +
+      chalk.green(this.name) + chalk.yellow(' command.')
```

Comment:
IMHO that should've been a separate commit, but I don't want to make this more complicated than it already is :wink: